.shellcheckrc

```diff
@@ -5,31 +5,3 @@
 # we need this because we specify one file at a time in order to trivially
 # detect which files are failing
 external-sources=true
-
-# Currently disabled these errors will take care of them later
-
-# Not following: (error message here)
-# This variable is assigned to itself, so the assignment does nothing.
-# https://github.com/koalaman/shellcheck/wiki/SC2269
-disable=SC2269
-# Reasons include: file not found, no permissions, not included on the command line, not allowing shellcheck to follow files with -x, etc.
-# https://github.com/koalaman/shellcheck/wiki/SC1091
-disable=SC1091
-# Since you double quoted this, it will not word split, and the loop will only run once
-# https://github.com/koalaman/shellcheck/wiki/SC2066
-disable=SC2066
-# Double quote to prevent globbing and word splitting
-# https://github.com/koalaman/shellcheck/wiki/SC2086
-disable=SC2086
```

Comment:
I see this disabled in many places in the code. It would be less code to disable it here, and this is where people will look for configuration. Are there any benefits from enabling it here, and disabling in code?

Reply:
Disabled only on places where we expect this situation. For example [here](https://github.com/kubernetes-sigs/kueue/blob/7f35d8da425ca8c35ea14a4dc982f48fa0d9ef02/hack/e2e-test.sh#L70) we should split `$GINKGO_ARGS` by spaces and allow user to put multiple args in variables. In another places (look at [commit](https://github.com/kubernetes-sigs/kueue/pull/2763/commits/7f35d8da425ca8c35ea14a4dc982f48fa0d9ef02)) we should prevent this situation.

Reply:
Thanks, this makes sense - so there are many more places which benefit from enabling it.